deployments/data-hub/data-hub--stg.yaml

```diff
@@ -190,9 +190,12 @@ spec:
       - name: aws-secret-volume
         mountPath: /home/airflow/.aws
         readOnly: true
-      - name: gmail-secret-volume
+      - name: gmail-production-secret-volume
         mountPath: /dag_secret_files/gmail/
```

Comment:
and rename the path maybe as well?

Reply:
To change the path, I think I should create new secret if I am correct?